### PR TITLE
293 first author resource list

### PR DESCRIPTION
**Pull request by pkdash**
Waiting to be merged.

### CODE REVIEW

File: hs_core/forms.py [hunk 1]
```diff
@@ -498,11 +498,13 @@ def __init__(self, *args, **kwargs):
 
 class PartyForm(ModelForm):
     def __init__(self, *args, **kwargs):
+        if 'initial' in kwargs:
+            if kwargs['initial']['description']:
```
Comment (ironfroggy) [inline hs_core/forms.py]: should you check that 'description' exists or catch the KeyError?

File: hs_core/hydroshare/utils.py [hunk 2]
```diff
@@ -310,15 +310,21 @@ def prepare_resource_default_metadata(resource, metadata, res_title):
     metadata.append({'date': {'type': 'created', 'start_date': resource.created}})
     metadata.append({'date': {'type': 'modified', 'start_date': resource.updated}})
 
-    if resource.creator.first_name:
-        first_creator_name = "{first_name} {last_name}".format(first_name=resource.creator.first_name,
-                                                                   last_name=resource.creator.last_name)
+    creator_data = {}
+    user_profile = get_profile(resource.creator)
+    user_full_name = resource.creator.get_full_name()
+    if len(user_full_name) > 0:
```
Comment (ironfroggy) [inline hs_core/hydroshare/utils.py]: you can just do `if user_full_name:` to test for non-empty strings

File: hs_core/forms.py (hunk 1 above)
Comment (pkdash) [inline hs_core/forms.py]: Is there a possibility of getting KeyError if the Party model has the 'description' attribute?

**Comment by pkdash** [comment]
@ironfroggy Is this ready for merge?

**Comment by ironfroggy** [comment]
+1

**Comment by pkdash** [comment]
@mjstealey Please merge this.

**Pull request merged**